sway/config/bar.c

```diff
@@ -91,6 +87,10 @@ struct bar_config *default_bar_config(void) {
 	bar->binding_mode_indicator = true;
 	bar->verbose = false;
 	bar->pid = 0;
+	if (!(bar->mode = strdup("dock"))) goto cleanup;
```

Comment:
Braces are mandatory

Reply:
Tell that to whoever wrote that if in the first place, but meh, sure

Reply:
Yeah, I saw that it was like this before too. But when we actually touch those lines, we might as well fix style issues.

Reply:
Yeah I'll fix it. You underestimate how much of a pain this whole serie of patch was though :P

Reply:
Yeah, I understand this is very tedious and painful work and I'm grateful you take the time to do it!

Reply:
It's done :) (force pushed after fixing each individual commits)